src/components/LandingPage.css

```diff
@@ -0,0 +1,18 @@
+
+.topBar {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+    align-content: baseline;
+    align-items: center;
+}
+
+.navigation ul, .logo{
+    display: flex;
+    flex-direction: row;
```

Comment:
I think row is the default flex-direction value, so you can leave it off if you want. Totally ok to include it though!